2020-01-29 14:07:27,916 ERROR com.sep.bank.service.Logging [https-jsse-nio-8450-exec-10] ERROR: Podaci koji su uneti za karticu nisu validni. Transaction: Transaction{id=1, amount=285.0, timestamp=2020-01-29 14:06:26.0, paymentId=1, paymentStatus=FAILURE, customer=com.sep.bank.model.Customer@28afd667}. Transaction: Transaction{id=1, amount=285.0, timestamp=2020-01-29 14:06:26.0, paymentId=1, paymentStatus=FAILURE, customer=com.sep.bank.model.Customer@28afd667}
2020-01-29 17:27:32,683 ERROR com.sep.bank.service.Logging [https-jsse-nio-8451-exec-6] ERROR: Podaci koji su uneti za karticu nisu validni. Transaction: Transaction{id=2, amount=285.0, timestamp=Wed Jan 29 17:27:32 CET 2020, paymentId=1, paymentStatus=FAILURE, customer=com.sep.bank.model.Customer@5f5a1aaa}. Transaction: Transaction{id=2, amount=285.0, timestamp=Wed Jan 29 17:27:32 CET 2020, paymentId=1, paymentStatus=FAILURE, customer=com.sep.bank.model.Customer@5f5a1aaa}
2020-01-29 17:36:09,149 ERROR com.sep.bank.service.Logging [https-jsse-nio-8451-exec-5] ERROR: Podaci koji su uneti za karticu nisu validni. Transaction: Transaction{id=2, amount=285.0, timestamp=Wed Jan 29 17:35:23 CET 2020, paymentId=1, paymentStatus=FAILURE, customer=com.sep.bank.model.Customer@41e4bf51}. Transaction: Transaction{id=2, amount=285.0, timestamp=Wed Jan 29 17:35:23 CET 2020, paymentId=1, paymentStatus=FAILURE, customer=com.sep.bank.model.Customer@41e4bf51}
2020-01-29 18:12:08,080 ERROR com.sep.bank.service.Logging [https-jsse-nio-8451-exec-5] ERROR: Podaci koji su uneti za karticu nisu validni. Transaction: Transaction{id=2, amount=285.0, timestamp=Wed Jan 29 18:12:07 CET 2020, paymentId=1, paymentStatus=FAILURE, customer=com.sep.bank.model.Customer@2e1cc12a}. Transaction: Transaction{id=2, amount=285.0, timestamp=Wed Jan 29 18:12:07 CET 2020, paymentId=1, paymentStatus=FAILURE, customer=com.sep.bank.model.Customer@2e1cc12a}
2020-01-29 18:17:15,430 ERROR com.sep.bank.service.Logging [https-jsse-nio-8451-exec-5] ERROR: Podaci koji su uneti za karticu nisu validni. Transaction: Transaction{id=2, amount=285.0, timestamp=Wed Jan 29 18:17:15 CET 2020, paymentId=1, paymentStatus=FAILURE, customer=com.sep.bank.model.Customer@3abfbf81}. Transaction: Transaction{id=2, amount=285.0, timestamp=Wed Jan 29 18:17:15 CET 2020, paymentId=1, paymentStatus=FAILURE, customer=com.sep.bank.model.Customer@3abfbf81}
2020-01-29 18:23:24,134 ERROR com.sep.bank.service.Logging [https-jsse-nio-8451-exec-5] ERROR: I/O error on PUT request for "https://localhost/8450/transfer-funds/1": Connection refused: connect; nested exception is java.net.ConnectException: Connection refused: connect. Transaction: Transaction{id=2, amount=285.0, timestamp=Wed Jan 29 18:23:22 CET 2020, paymentId=1, paymentStatus=PROCESSING, customer=com.sep.bank.model.Customer@6da355c2}
2020-01-29 18:54:53,499 ERROR com.sep.bank.service.Logging [https-jsse-nio-8451-exec-5] ERROR: I/O error on PUT request for "https://localhost/8450/transfer-funds/1": Connection refused: connect; nested exception is java.net.ConnectException: Connection refused: connect. Transaction: Transaction{id=2, amount=285.0, timestamp=Wed Jan 29 18:54:51 CET 2020, paymentId=1, bankType=ISSUER, paymentStatus=PROCESSING, customer=com.sep.bank.model.Customer@682c281f}
2020-01-29 18:59:21,719 ERROR com.sep.bank.service.Logging [https-jsse-nio-8451-exec-9] ERROR: 404 null. Transaction: Transaction{id=2, amount=285.0, timestamp=Wed Jan 29 18:59:21 CET 2020, paymentId=1, bankType=ISSUER, paymentStatus=PROCESSING, customer=com.sep.bank.model.Customer@5414f121}
